account_credit_control/wizard/credit_control_communication.py

```diff
@@ -208,7 +203,8 @@ def _generate_report(self):
 
         """
         report_name = 'account_credit_control.report_credit_control_summary'
-        return self.env['report'].get_pdf(self, report_name)
+        report = self.env.ref(report_name)
+        return report.render_qweb_pdf()
```

Comment:
Thank you @kenvac ! 
I'm wondering if the method `def _generate_report()` is really necessary. It is not called from the rest of the module and it is not present in the standard Odoo also.